modules/openapi-generator/src/main/java/org/openapitools/codegen/CodegenSecurity.java

```diff
@@ -40,7 +40,7 @@ public class CodegenSecurity {
     public String keyParamName;
     public Boolean isKeyInQuery, isKeyInHeader, isKeyInCookie;
     // Oauth specific
-    public String flow, authorizationUrl, tokenUrl, refreshUrl;
+    public String flow, authorizationUrl, tokenUrl, refreshUrl, openIdConnectUrl;
```

Comment:
`openIdConnectUrl` is already defined in the same class
```suggestion
    public String flow, authorizationUrl, tokenUrl, refreshUrl
```